tests/network/vmi_networking.go

```diff
@@ -684,7 +684,13 @@ var _ = Describe(SIG("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:
 			configureIpv6 := func(vmi *v1.VirtualMachineInstance) error {
 				networkCIDR := api.DefaultVMIpv6CIDR
 
-				err := console.RunCommand(vmi, "dhclient -6 eth0", 30*time.Second)
+				err := console.RunCommand(
+					vmi,
+					"nmcli con modify \"$(nmcli -g NAME con show)\" ipv6.method dhcp", 5*time.Second)
+				if err != nil {
+					return err
+				}
+				err = console.RunCommand(vmi, "nmcli con up \"$(nmcli -g NAME con show)\"", 30*time.Second)
```

Comment:
Could you please elaborate what are the advantages of manually configuring the network manager over configuring it via cloud-init?

We could enable the  DHCPv6 client using:
https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#dhcp6-bool

Routes and gateway (defined bellow) could be configured using:
https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#routes-sequence-of-mapping
https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#gateway4-or-gateway6-scalar

Reply:
> Could you please elaborate what are the advantages of manually configuring the network manager over configuring it via cloud-init?
> 
> We could enable the DHCPv6 client using: https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#dhcp6-bool

It configures it as method `auto` and not `dhcp`.
 
> Routes and gateway (defined bellow) could be configured using: https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#routes-sequence-of-mapping https://cloudinit.readthedocs.io/en/latest/reference/network-config-format-v2.html#gateway4-or-gateway6-scalar

Because we cannot configure the correct method, we cannot configure this one as well.